windows-iotcore/TOC.md

```diff
@@ -1,4 +1,5 @@
 # [Windows IoT Core documentation](index.md)
+# [Windows 10 IoT Enterprise](learn-about-hardware/WindowsIoTEnterprise.md)
```

Comment:
this will not work. You need to create a separate folder or the breadcrumbs will break.